### PR TITLE
fix/ignore standardjs linter errors and add to travis

### PR DESCRIPTION
**Pull request by spresnal**
Fixed or ignored all the errors that came up when running standard. We'll need to revisit some of the ignored lines & test files to get the ignored errors resolved (ideally separate issues to get them corrected). However, this should be a good step in the right direction because we can get Travis running the linter for new commits.

I'll comb through this commit again a little later to double check all the changes, but should be close to what we need.

### CODE REVIEW

File: client/app/src/accounts/account.controller.js [hunk 1]
```diff
@@ -490,16 +486,16 @@
         }
       }, 200)
 
-      self.exchangeSellTransaction = transaction
-      changerService.monitorExchange(resp).then(
+      self.exchangeSellTransaction = transaction // eslint-disable-line no-undef
```
Comment (spresnal) [inline client/app/src/accounts/account.controller.js]: These two lines appeared to be the artifact of some refactoring. Wasn't sure if it was appropriate to remove them or not.

File: client/app/src/accounts/account.controller.js [hunk 2]
```diff
@@ -1229,14 +1225,14 @@
...
-      if (self.network.themeDark) {self.currentTheme = 'dark'}
+      if (self.network.themeDark) { self.currentTheme = 'dark' }
     }
 
     // Compare vibrant colors from image with default material palette
     // And returns the most similar primary and accent palette
     function generateDynamicPalette (callback) {
       if (!self.network.background) {
-        callback(false)
+        callback(false) // eslint-disable-line standard/no-callback-literal
```
Comment (spresnal) [inline client/app/src/accounts/account.controller.js]: Wasn't sure what the appropriate pattern to use here was so these got ignored

File: client/app/src/components/account/signing-tab.controller.js [hunk 3]
```diff
@@ -28,14 +28,13 @@
           .clickOutsideToClose(true)
           .title(message)
           .ariaLabel(message)
-          .theme(self.currentTheme)
+          .theme(self.currentTheme) // eslint-disable-line no-undef
```
Comment (spresnal) [inline client/app/src/components/account/signing-tab.controller.js]: No reference to 'self' here so this appears to be an issue that needs fixing

File: client/app/src/components/account/signing-tab.controller.js [hunk 4]
```diff
@@ -61,7 +60,7 @@
             $mdDialog.hide()
           },
           function (error) {
-            accountCtrl.showMessage(error)
+            accountCtrl.showMessage(error)  // eslint-disable-line no-undef
```
Comment (spresnal) [inline client/app/src/components/account/signing-tab.controller.js]: No reference to 'accountCtrl' here so this appears to be an issue that needs fixing as well

**Comment by alexbarnsley** [comment]
All looking good to me, but I'm no StandardsJS expert! Everything seems to be working as expected though 👌

**Comment by spresnal** [comment]
@j-a-m-l I resolved new merge conflicts. Please give this a review when you are available.

**Comment by alexbarnsley** [comment]
@spresnal done a test of a few key areas - can't see any issues! Nice work 😄 

+5! 👌

**Pull request merged**